.openshift-ci/crawler/repackage.sh

```diff
@@ -8,8 +8,15 @@ source .openshift-ci/crawler/env.sh
 source .openshift-ci/crawler/setup-staging.sh
 
 mkdir -p .build-data/cache
+
+echo "Prepare cache, ${KERNEL_BUNDLE_BUCKET}/cache.yml..."
+gsutil cp "${KERNEL_BUNDLE_BUCKET}/cache.yml" .build-data/cache/cache.yml || true
 touch .build-data/cache/cache.yml
```

Comment:
do we still need this `touch`?

Reply:
It's mostly paranoia in case if the `cache.yml` is not found in the bucket or something similar has happened.